boilerplate/app/i18n/hi.ts

```diff
@@ -0,0 +1,131 @@
+    letsGo: "चलो चलते हैं!", // @demo remove-current-line
+  },
+  errorScreen: {
+    title: "कुछ गलत हो गया!",
+    friendlySubtitle:
+      "यह वह स्क्रीन है जो आपके उपयोगकर्ता प्रोडक्शन में देखेंगे जब कोई त्रुटि होगी। आप इस संदेश को कस्टमाइज़ करना चाहेंगे (जो `app/i18n/hi.ts` में स्थित है) और शायद लेआउट भी (`app/screens/ErrorScreen`). यदि आप इसे पूरी तरह से हटाना चाहते हैं, तो `app/app.tsx` में <ErrorBoundary> कंपोनेंट की जांच करें।",
+    reset: "ऐप रीसेट करें",
+    traceTitle: "%{name} स्टैक से त्रुटि", // @demo remove-current-line
+  },
+  emptyStateComponent: {
+    generic: {
+      heading: "इतना खाली... इतना उदास",
+      content:
+        "अभी तक कोई डेटा नहीं मिला। रीफ्रेश करने या ऐप को पुनः लोड करने के लिए बटन पर क्लिक करें।",
+      button: "चलो फिर से कोशिश करते हैं",
+    },
+  },
+  // @demo remove-block-start
+  errors: {
+    invalidEmail: "अमान्य ईमेल पता।",
+  },
+  loginScreen: {
+    logIn: "लॉग इन करें",
+    enterDetails:
+      "टॉप सीक्रेट जानकारी को अनलॉक करने के लिए नीचे अपना विवरण दर्ज करें। आप कभी अनुमान नहीं लगा पाएंगे कि हमारे पास क्या इंतजार कर रहा है। या शायद आप कर सकते हैं; यह रॉकेट साइंस नहीं है।",
+    emailFieldLabel: "ईमेल",
+    passwordFieldLabel: "पासवर्ड",
+    emailFieldPlaceholder: "अपना ईमेल पता दर्ज करें",
+    passwordFieldPlaceholder: "सुपर सीक्रेट पासवर्ड यहाँ",
+    tapToLogIn: "लॉग इन करने के लिए टैप करें!",
+    hint: "संकेत: आप किसी भी ईमेल पते और अपने पसंदीदा पासवर्ड का उपयोग कर सकते हैं :)",
+  },
+  demoNavigator: {
+    componentsTab: "कंपोनेंट्स",
+    debugTab: "डीबग",
+    communityTab: "समुदाय",
+    podcastListTab: "पॉडकास्ट",
+  },
+  demoCommunityScreen: {
+    title: "समुदाय से जुड़ें",
+    tagLine:
+      "Infinite Red के React Native इंजीनियरों के समुदाय से जुड़ें और हमारे साथ अपने ऐप विकास को बेहतर बनाएं!",
+    joinUsOnSlackTitle: "Slack पर हमसे जुड़ें",
+    joinUsOnSlack:
+      "क्या आप चाहते हैं कि दुनिया भर के React Native इंजीनियरों से जुड़ने के लिए कोई जगह हो? Infinite Red Community Slack में बातचीत में शामिल हों! हमारा बढ़ता हुआ समुदाय प्रश्न पूछने, दूसरों से सीखने और अपने नेटवर्क को बढ़ाने के लिए एक सुरक्षित स्थान है।",
+    joinSlackLink: "Slack समुदाय में शामिल हों",
+    makeIgniteEvenBetterTitle: "Ignite को और बेहतर बनाएं",
+    makeIgniteEvenBetter:
+      "Ignite को और बेहतर बनाने का कोई विचार है? हमें यह सुनकर खुशी होगी! हम हमेशा ऐसे लोगों की तलाश में रहते हैं जो हमें सर्वश्रेष्ठ React Native टूलिंग बनाने में मदद करना चाहते हैं। Ignite के भविष्य को बनाने में हमारे साथ शामिल होने के लिए GitHub पर हमसे जुड़ें।",
+    contributeToIgniteLink: "Ignite में योगदान दें",
+    theLatestInReactNativeTitle: "React Native में नवीनतम",
+    theLatestInReactNative: "हम आपको React Native के सभी प्रस्तावों पर अपडेट रखने के लिए यहां हैं।",
+    reactNativeRadioLink: "React Native रेडियो",
+    reactNativeNewsletterLink: "React Native न्यूजलेटर",
+    reactNativeLiveLink: "React Native लाइव",
+    chainReactConferenceLink: "Chain React कॉन्फ्रेंस",
```

Comment:
Conference -> "SammElan"

Reply:
The use of "कॉन्फ्रेंस" is quite common in Hindi. It keeps the translation aligned with the way people speak in bilingual or Hindi context.

Reply:
Okay then